src/Processors/Formats/Impl/NativeORCBlockInputFormat.cpp

```diff
@@ -137,12 +154,25 @@ static DataTypePtr parseORCType(const orc::Type * orc_type, bool skip_columns_wi
             return std::make_shared<DataTypeDateTime64>(9);
         case orc::TypeKind::TIMESTAMP_INSTANT:
             return std::make_shared<DataTypeDateTime64>(9, "UTC");
+        case orc::TypeKind::CHAR:
         case orc::TypeKind::VARCHAR:
         case orc::TypeKind::BINARY:
-        case orc::TypeKind::STRING:
-            return std::make_shared<DataTypeString>();
-        case orc::TypeKind::CHAR:
-            return std::make_shared<DataTypeFixedString>(orc_type->getMaximumLength());
+        case orc::TypeKind::STRING: {
```

Comment:
Put `{` on a new line.

Reply:
But it is the formatting result of `clang-format`